examples/docs/en-US/select.md

```diff
@@ -665,6 +665,7 @@ If the binding value of Select is an object, make sure to assign `value-key` as
 | multiple | whether multiple-select is activated | boolean | — | false |
 | disabled | whether Select is disabled | boolean | — | false |
 | value-key | unique identity key name for value, required when value is an object | string | — | value |
+| label-key | unique identity key name for label, required when value is an object | string | — | label |
```

Comment:
- I think this attribute should not be required. Only when the label doesn't display, the attribute is needed. You can add a tip for this.
- Just copy these change to Chinese and Spanish docs.